README.md

```diff
@@ -22,6 +22,12 @@ Think of it like [Tailwind](https://tailwindcss.com/) for JavaScript.
 
 That's it. It will initialize itself.
 
+For production environments, it's recommended to pin a specific version number in the link to avoid unexpected breakage from newer versions. 
```

Comment:
I'd probably add some bold text here too so it's a bit more obvious.

Reply:
We can do, although people won't read the readme anyway. 😂
I'll leave it with Caleb if he wants it bold or not or he wants to reword it. To me it just needs to be thee and it's already okay since it's not part of a very long paragraph (same as Vue's documentation). 👍